client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -17,7 +17,7 @@ import {
 } from '../../../resources/aria-at-test-io-format.mjs';
 import { TestWindow } from '../../../resources/aria-at-test-window.mjs';
 import { evaluateAtNameKey } from '../../../utils/aria.js';
-import commandsJson from '../../../resources/commands.json';
+import commandsJson from '../../../resources/commandsV1.json';
```

Comment:
This is a v2 commands file. Files that look like the [following](https://github.com/w3c/aria-at/blob/master/tests/commands.json) is v2